src/models/classes/customer.js

```diff
@@ -9,7 +9,6 @@
     {
         this.name = this.generateName();
         this.order = new CustomerOrder(products);
-        this.timestamp = GAME.model.config.hour;
```

Comment:
And then pass timestamp to the constructor when generating the order (I think in `CustomerController.generateOrder`), when the Customer object is created.